### PR TITLE
Upgrade PHP 7.1 + upgrade dependencies, #21

### PR DESCRIPTION
**Pull request by Awkan**
#21 

- Remove PHP 5.6 utilisisability in favor of PHP7.1
- Upgrade dependencies (newer + explicitly compatible with PHP7+)
- Remove compatibility with SF2 (minimum requirement is now SF 3.4+)
- Update Travis config

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 language: php
 
 php:
-    - 5.6
-    - 7.0
     - 7.1
     - 7.2
```
Comment (Nek-) [inline .travis.yml]: `- nightly` ? :)

File: .travis.yml [hunk 2]
```diff
@@ -12,18 +10,17 @@ cache:
...
           env: 'COMPOSER_FLAGS="--prefer-lowest --prefer-stable"'
+        - php: 7.2
+          env: 'COMPOSER_FLAGS="--prefer-lowest --prefer-stable"'
 
 branches:
     only:
         - master
         - 1.x
         - 2.x
+        - 3.x
```
Comment (Nek-) [inline .travis.yml]: On va rester sur master pour l'instant.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Changed
 - Upgrade `php-cs-fixer` package from `^1.11` to `^2.2` && update `.php_cs` rules
```
Comment (Nek-) [inline CHANGELOG.md]: `2.11` du coup

**Comment by Awkan** [comment]
@Nek- Branch is now up to date 😉

**Comment by Awkan** [comment]
@Nek- The usage of `nightly` option make travis tests fail (due to PHP version requirement for package `phpspec`).

What do you wanna do ? 😃

**Pull request merged**